src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -93,6 +94,14 @@ private enum QueueState {
 
     private Map<QueueOperation, QueueAction> queueActions = new HashMap<>();
 
+    private Map<String, DequeueStatistic> dequeueStatistic = new ConcurrentHashMap<>();
+
+    public class DequeueStatistic{
```

Comment:
Why not making this part of the existing QueueStatistics instead of another one here?

Reply:
The idea was to provide this to the UI of our  communication server that lists the queues. Let me see if we can also easily provide these values through the QueueStatisticsHandler / QueueStatisticsCollector